app/templates/travisci-vs-jenkins/index.hbs

```diff
@@ -192,8 +192,8 @@
   <UiKit::Box
     @tag='section'
     @borderWidth={{hash top='xs'}}
-    @margin={{hash bottom=16}}
-    @padding={{hash top=20}}
+    @margin={{hash bottom=24}}
+    @padding={{hash top=24}}
```

Comment:
Once I saw the whole page altogether, I felt this section could use a little more spacing